README.md

```diff
@@ -91,12 +91,15 @@ following form:
 }
 ```
 
-Please note: implementations may have security using oauth2 or zcaps, but not both.
-Implementations may also contain no security (do not add a OAUTH2 or ZCAP section in that case).
+Please note: implementations may have security using oauth2 or zcaps, but not
+both.
+Implementations may also contain no security (do not add a OAUTH2 or ZCAP
+section in that case).
 
-#### Testing locally
+### Testing locally
 
-If you need to add implementations for endpoints running locally you may add a config file in the root dir of your test project:
+If you need to add implementations for endpoints running locally you may add a
+config file in the root dir of your test project:
```

Comment:
Is there anyway to test _only_ a locally running implementation? If so (or not), we should likely handle it as a separate issue/PR...but I keep wondering, so thought I'd ask.

Ideally, something short of removing all the other implementations from the list temporarily...

Reply:
Yes, there is. In the test suites you can add a config file `.vcApiTestImplementationsConfig.cjs` and then you can add your implementations running locally like this -
https://github.com/w3c-ccg/vc-api-test-suite-implementations/blob/main/README.md#testing-locally and update the tags in the tests to match the tag you specify in your config. And then run the test like you would run normally.

Reply:
Included those additional guidelines under the `Testing locally` section to make it clearer [168dbd1](https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/91/commits/168dbd137004b5962a04517d88c695e3d5c13278)

Reply:
@JSAssassin this is great! Can you expand on how this is done, though?
> update the tags in the tests to match the tag you specify in your config. And then run the test like you would run normally

Reply:
@BigBlueHat Updated the section and added example here - [03fc824](https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/91/commits/03fc824d1000666761ec52fcff4c997db6469e5a).